mne/annotations.py

```diff
@@ -567,6 +567,49 @@ def crop(self, tmin=None, tmax=None, emit_warning=False, verbose=None):
 
         return self
 
+    @verbose
+    def set_durations(self, mapping, verbose=None):
+        """Set annotation durations(s). Operates inplace.
```

Comment:
maybe remind somewhere that durations are in seconds.

Reply:
```suggestion
        """Set annotation duration(s). Operates inplace.
```